scripts/start.sh

```diff
@@ -74,7 +74,9 @@ while [[ ! $(docker-compose logs connect) =~ "Herder started" ]]; do
   fi
 done
 
-if [[ ! $(docker-compose exec connect timeout 3 nc -zv irc.wikimedia.org 6667) =~ "open" ]]; then
+docker-compose exec connect timeout 3 nc -zv irc.wikimedia.org 6667
+
+if [[ $? != 0 ]]; then
   echo -e "\nERROR: irc.wikimedia.org 6667 is unreachable. Please ensure connectivity before proceeding or try setting 'irc.server.port' to 8001 in scripts/connectors/submit_wikipedia_irc_config.sh\n"
   exit 1
 fi
```

Comment:
Thanks @elismaga , for your consideration you could consider using the OR operator here to more explicitly tie the return code to the failure case.  It's slightly more explicit and will prevent the possibility that someone accidentally sneaks a command in front of the conditional and failure block.

```suggestion-5
docker-compose exec connect timeout 3 nc -zv irc.wikimedia.org 6667 || {
  echo -e "\nERROR: irc.wikimedia.org 6667 is unreachable. Please ensure connectivity before proceeding or try setting 'irc.server.port' to 8001 in scripts/connectors/submit_wikipedia_irc_config.sh\n"
  exit 1
}
```

Reply:
@rspurgeon Thanks for the suggestion.